contracts/CandidateHub.template

```diff
@@ -157,12 +157,10 @@ contract CandidateHub is ICandidateHub, System, IParamSubscriber {
   function jailValidator(address operateAddress, uint256 round, uint256 fine) 
         external override onlyValidator onlyIfCandidateAddress(operateAddress) {
     uint256 indexPlus1 = operateMap[operateAddress];
-    if (indexPlus1 == 0) return;
-
     uint index_ = indexPlus1 - 1;
     Candidate storage c = candidateSet[index_];
     uint256 margin = c.margin;
-    if (margin >= dues && margin - dues >= fine) { //@openissue
+    if (margin >= dues + fine) {
```

Comment:
@jackcrypto9527 like this

Reply:
1. dues will always have a contained value since it's it is tested to be <= cap before any update
2. if worst happens and an evil validator will call jailValidator() with say fine = MAX_UINT-1,  then 'dues + fine' will 
    auto-revert (as we want it to!) since it's not inside an 'unchecked' block and no harm done
3. note that margin deduction (see: margin - fine) can also overflow 
4. and there too (and in many other places..) we rely on revert-on-overflow protection

now this is a small thing and will work or fail either way. my point is that we do not want to get into the habit of complicating our code to protect against issues guaranteed to be protected by the language itself